### PR TITLE
Document OpenLineage plugin in 449 release note

### PR DESCRIPTION
**Pull request by ebyhr**
## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

**Comment by ebyhr** [comment]
The following failure will be fixed in https://github.com/trinodb/trino/pull/22223:
```
Error:    TestTestSetup.testSetupOfTests:45 Errors: 
		Missing dependency on io.airlift:junit-extensions in /home/runner/work/trino/trino/plugin/trino-openlineage/pom.xml
```

File: docs/src/main/sphinx/release/release-449.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## General
 
+* Add [OpenLineage event listener plugin](/admin/event-listeners-openlineage). ({issue}`21265`)
 * Add an event listener which exposes collected events to an HTTP endpoint. ({issue}`22158`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-449.md:6]: What is this? I think this is wrong and a cut and paste error @colebow  .. right .. the HTTP event listener is old - https://trino.io/docs/current/admin/event-listeners-http.html

Comment (mosabua) [reply docs/src/main/sphinx/release/release-449.md:6]: Oh .. it is separate ... but definitely needs docs as well

File: docs/src/main/sphinx/release/release-449.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## General
 
+* Add [OpenLineage event listener plugin](/admin/event-listeners-openlineage). ({issue}`21265`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-449.md:5]:
```suggestion
* Add [OpenLineage event listener](/admin/event-listeners-openlineage). ({issue}`21265`)
```

**Review by mosabua — APPROVED** [review]

**Pull request merged**